### PR TITLE
Remove made up copyright date

### PR DESCRIPTION
**Pull request by userappgate**

### CODE REVIEW

File: cmd/help.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 func HelpTemplate() string {
 	cobra.AddTemplateFunc("now", time.Now)
 	cobra.AddTemplateFunc("caller", getCaller)
-	return `© {{ now.Year }} Appgate Cybersecurity, Inc.
+	return `© Appgate Cybersecurity, Inc.
```
Comment (kajes) [inline cmd/help.go:16]:
```suggestion
	return `© 2022-{{ now.Year }} Appgate Cybersecurity, Inc.
```
This is better.

Comment (userappgate) [reply cmd/help.go:16]: it's incorrect. Copyright is about when it was written, not when it's being executed.

File: cmd/help.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ import (
 func HelpTemplate() string {
 	cobra.AddTemplateFunc("now", time.Now)
```
Comment (danijeel) [inline cmd/help.go:14]:
```suggestion
```

**Review by danijeel — APPROVED** [review]
Removed un-used template variable  `cobra.AddTemplateFunc("now", time.Now)`

File: cmd/help.go (hunk 1 above)
Comment (kajes) [reply cmd/help.go:16]: It's also supposed to guide when the copyright may expire, which it doesn't do if you remove the year entirely. I don't think I've seen a copyright like this without a year provided.

Comment (userappgate) [reply cmd/help.go:16]: How can it guide on when the copyright expires if it always shows today's date?

Comment (kajes) [reply cmd/help.go:16]: Just leave `2022` there, and an indicator that is wrong for a short time every year is absolutely better than nothing at all

Comment (kajes) [reply cmd/help.go:16]: actually, just print 2022-2023. By the time we release the next version, it's going to be correct

**Pull request merged**